docs/user/pings/metrics.md

```diff
@@ -1,29 +1,36 @@
 # The `metrics` ping
 
 ## Description
-The `metrics` ping is intended for all of the metrics that are explicitly set by the application or are included in the application's `metrics.yaml` file (except events).
-The reported data is tied to the ping's *measurement window*, which is the time between the collection of two `metrics` ping.
-Ideally, this window is expected to be about 24 hours, given that the collection is scheduled daily at 4AM.
-Data in the [`ping_info`](index.md#the-ping_info-section) section of the ping can be used to infer the length of this window.
+The `metrics` ping is intended for all of the metrics that are explicitly set by the application or are included in the application's `metrics.yaml` file (except events). 
+The reported data is tied to the ping's *measurement window*, which is the time between the collection of two `metrics` ping. 
+Ideally, this window is expected to be about 24 hours, given that the collection is scheduled daily at 4AM. 
+However, the metrics ping is only sent while the application is actually running, so in practice, it may not meet the 4AM target very frequently.
```

Comment:
```suggestion
However, the metrics ping is only submitted while the application is actually running, so in practice, it may not meet the 4AM target very frequently.
```